app/presenters/sufia/work_show_presenter.rb

```diff
@@ -42,6 +42,10 @@ def stats_path
       Sufia::Engine.routes.url_helpers.stats_work_path(self)
     end
 
+    def member_presenters(ids = ordered_ids, presenter_class = composite_presenter_class)
+      super.delete_if { |presenter| current_ability.cannot?(:read, presenter.solr_document) }
```

Comment:
@hortongn While that fix works for PSU where we have few files in the works at present, it may not work for every implementation.  I agree with @atz that a SearchBuilder or a SolrDocument.where query would be best for everyone and likely PSU would need to replace the code we have in the future.  I think the "better" fix would be to include hydra-access-controls into this query: https://github.com/projecthydra/curation_concerns/blob/master/app/presenters/curation_concerns/work_show_presenter.rb#L132